security/compliance_operator/co-management/compliance-operator-installation.adoc

```diff
@@ -10,7 +10,7 @@ Before you can use the Compliance Operator, you must ensure it is deployed in th
 
 [IMPORTANT]
 ====
-The Compliance Operator might report incorrect results on managed platforms, such as OpenShift Dedicated, Red Hat OpenShift Service on AWS, and Microsoft Azure Red Hat OpenShift. For more information, see the link:https://access.redhat.com/solutions/6983418[Red Hat Knowledgebase Solution #6983418].
+The Compliance Operator might report incorrect results on managed platforms, such as OpenShift Dedicated and Microsoft Azure Red Hat OpenShift. For more information, see the link:https://access.redhat.com/solutions/6983418[Red Hat Knowledgebase Solution #6983418].
```

Comment:
@rhmdnd I am wondering if we still have the same issue on classic ROSA - Red Hat OpenShift Service on AWS. Should be keep the "Red Hat OpenShift Service on AWS"?

Reply:
Good call - fixed.